packages/template-retail-react-app/app/components/swatch-group/index.test.js

```diff
@@ -108,7 +110,7 @@ describe('Swatch Component', () => {
         const history = createMemoryHistory()
         history.push('/en-GB/swatch-example?color=JJ2XNXX')
 
-        render(
+        renderWithProviders(
```

Comment:
Why do we need to use renderWithProviders? I think it was intentional to not use the it since there is not need to render other providers to test this component.